### ISSUE TITLE
A version number of the WaziCloud should be shown on the top page 

### ISSUE THREAD

**Issue by SvenZeisberg** [Waziup/WaziCloud#256]
It would be good, if a version number of the wazi-cloud would be shown on the left side menu lower end of page.

**Comment by cdupont** [comment]
Good idea. How this versioning should work? Probably an independent number for the full WaziCloud is better.

**Comment by cdupont** [comment]
I could put a .env file in the dashboard repo. It would be easy to read and display it.
